lib/node/Creator.js

```diff
@@ -71,6 +71,13 @@ const NodeCreator = {
   nthRoot (radicandNode, rootNode) {
     const symbol = NodeCreator.symbol('nthRoot');
     return new math.expression.node.FunctionNode(symbol, [radicandNode, rootNode]);
+  },
+  powerTerm (base, exponent) {
```

Comment:
i only see you using it in two places, where you could just be creating a ^ node instead

am I missing something?